### ISSUE TITLE
Scales, bulks, cuts: Do they work? The answer might surprise you*

### ISSUE THREAD

**Issue by CodeXmonk** [CodeXmonk/topics#2]
Quality is more important than quantity. One home run is much better than two doubles. — Steve Jobs

##TL;DR:: Individual bioimpedance measurements of body fat percentage can vary around ~2% based on hydration levels (based on a Withings Smart Body Analyzer scale). The distribution of water in your body (the time it takes for everything to come into equilibrium after laying down for a long time) also results in a potential shift of approximately 1.5%. Individual measurements based on bioimpedance are not particularly accurate, but on average the reading is correct and long-term trends can be distinguished. A bulk/cut cycle can be an effective strategy for changing the composition of your body, if you proportion the cutting and bulking periods correctly.
**A high quality scale that measures body fat as well as weight can be an invaluable tool for tracking what works for both weight loss as well as muscle gain, and allows you to adapt and change your routine as you find out what works and what doesn’t.**

###Article link
https://www.instapaper.com/read/1340045846
###Author
Tyler Morgan-Wall
Tyler is a writer, programmer, physicist, and data scientist.

Key Takeaways

    parts of the articles that you find useful.

Useful Code Snippets

    useful codes that you want to save for later

Useful Tools

    useful tools that you want to try out or integrate into your workflow

Comments/Questions

    Your thoughts or questions on the articles.

Image (optional)

    give you the visual description of what the article about
